core-libs/setup/ssr/logger/loggers/legacy-server-logger.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ExpressServerLogger,
+  ExpressServerLoggerContext,
+} from './express-server-logger';
+
+/**
+ * @deprecated since 6.2, will be removed in 7.0 as contextual logging will be enabled by default.
```

Comment:
[MINOR]

Let's not promise a 7.0 version, as the policy of releases might change (e.g. we might potentially limit the number of breaking changes in 7.0, and only upgrade Angular. nothing decided yet though)
```suggestion
 * @deprecated since 6.2, will be removed in a new major version, as contextual logging will be enabled by default.
```